### DISCUSSION TITLE
中大語言課修課經驗(5)+求推薦英文課

### DISCUSSION THREAD

**Discussion by THH1118** [1101-GS4719-C/Python-Community-GS4719-C#1758] [General]
英語口語訓練 朱琼芝 老師

因為是口語訓練，所以這堂課就是會需要學生多講英文，所以老師每堂課都會隨機分組，上課期間就會讓你去跟組員分享或討論老師給的問題，所以課堂上其實不會很累，但這堂課的作業非常的多，基本上每周都有，也會定期口語小考，就是會給個題目然後自己寫稿背給老師聽，老師會要求這個搞裡面要用到特定的單字之類的，然後還有錄音作業跟期末報告，回家後會需要做蠻多事的，如果作業有按照老師的規則來做的話都會有好的成績，沒有的話當然就…，上完一學期後自己的確有感覺自己的口說有進步。

然後因為下學期想再修一堂英文課，所以想在這裡問問有沒有推薦的英文課，分數不用高沒關係主要是想學東西，好像在FB上看到很多人都推薦去上秦老師的課!

**Comment by WeiYen1103** [comment]
我之前大二修了英文與人生，我覺得這堂課的老師人很好，蠻溫柔的(不要踩到底線)，但是這堂課偏硬，每周都會要寫心得，還要發表文章，也會需要上台演講，要背稿(記得那時候我光寫稿就想了好久)，但真的可以學到蠻多東西了，也訓練了自己上台的能力。

**Comment by Chengbosco** [comment]
秦老師的課很好欸，EAP課是六人一組的報告，每兩人又再一小組，可以自由選擇你要報告的主題，可以很水也可以很有趣。ESP的話要決定每周要做什麼和玩什麼，包括但不限於文章心得、玩遊戲、影集心得、時事報告。但他上課就會一直試圖引導我們，也會一直加分，是很有趣，會學習如何發表，但實際上到的課就比較少(不是會硬塞字叫你背的那種老師)，期末考就得自己讀了。

**Comment by mrweixd** [comment]
正好我自認缺乏英文口說能力，上這種課確實不錯，就是不知道要不要背稿，我一向不擅長背誦。

**Comment by 12Jesse24** [comment]
這學期剛好有幸修到朱老師開的科技英文演講與剪報，真的可以學到很多可以在做英文簡報時使用的技巧，正如上文所說。每個禮拜都有作業，每次上課也有課堂練習，需要同學上台演講，然後台下的同學與老師負責評分，佔學期成績20%。剛開始真的蠻吃力的，尤其是對於我這種英文能力剛進來只有初級班的，到後面我有覺得越來越上手，甚至老師都有說我進步很多，真的很推英文能力不好的人去修，但分數就...不太樂觀，畢竟多半都是靠自己的實力取得的。
